src/agent/reqwest-retry/src/lib.rs

```diff
@@ -169,19 +169,20 @@ impl SendRetry for reqwest::RequestBuilder {
     }
 }
 
-#[cfg(test)]
-mod test {
-    use super::*;
+// TODO: convert to feature-gated integration test.
```

Comment:
Instead of commenting these out, can we use `#[ignore]`

Reply:
Ah, good call!